migrations/20200425013627-change-tags-to-lowercase.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+import Promise from 'bluebird';
+
+/** Both convertTagsToLowerCase and convertTagsToLowerCase return tags
+ * as comma separated values so the string_to_array function can be used in
+ * the update query */
+
+const convertTagsToLowerCase = objectWithTags => objectWithTags.tags.map(tag => tag.toLowerCase()).join(',');
+
+const convertTagsToUpperCase = objectWithTags => objectWithTags.tags.map(tag => tag.toUpperCase()).join(',');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const conversations = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Conversations"
+        WHERE "tags" IS NOT NULL;
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    await Promise.map(conversations, conversation =>
+      queryInterface.sequelize.query(
+        `
+        UPDATE "Conversations" c
+        SET "tags" = string_to_array(:tags_as_string, ',')
+        WHERE c.id = :id
+      `,
+        {
+          replacements: {
+            id: conversation.id,
+            tags_as_string: convertTagsToLowerCase(conversation),
```

Comment:
A pure SQL solution using [`LOWER`](https://w3resource.com/PostgreSQL/lower-function.php) would be more stable and performant, did you consider using it?

Reply:
No, I didn't consider using `LOWER`. I will update the query.